Parsedown.php

```diff
@@ -75,6 +75,27 @@ function setUrlsLinked($urlsLinked)
 
     protected $urlsLinked = true;
 
+    function setSafeLinksEnabled($safeLinksEnabled)
+    {
+        $this->safeLinksEnabled = $safeLinksEnabled;
+
+        return $this;
+    }
+
+    protected $safeLinksEnabled = true;
```

Comment:
IMO this shouldn't be enabled by default, it breaks BC. Enabling this without `setMarkupEscaped(true)` makes no sense anyway - it prevents absolutely nothing without escaped markup. Never confuse people by letting them do pointless things.

How about something like this (disallow enabling `safeLinksEnabled` without `markupEscaped` and always enable/disable `safeLinksEnabled` and `markupEscaped` at the same time; however, disabling `safeLinksEnabled` when `markupEscaped` is enabled is still possible - escaping markup doesn't necessarily imply a "safe mode", but in contrast, a "safe mode" isn't possible without escaping markup)?

```php
    function setMarkupEscaped($markupEscaped)
    {
        $this->markupEscaped = (bool) $markupEscaped;
        $this->safeLinksEnabled = (bool) $markupEscaped;
        return $this;
   }

    function setSafeLinksEnabled($safeLinksEnabled)
    {
        // or throw a exception instead
        $this->safeLinksEnabled = $this->markupEscaped && $safeLinksEnabled;
        return $this;
    }
```